.github/workflows/chainloop.yml

```diff
@@ -26,53 +29,45 @@ jobs:
           sparse-checkout: |
             .chainloop.yml
 
-      - uses: sigstore/cosign-installer@v3.1.1
-      
-      - name: Install Chainloop CLI
+      - name: Install Chainloop CLI, Labs and Cosign
         run: |
-          if [ -n "${CHAINLOOP_VERSION}" ]; then
-             curl -sfL https://docs.chainloop.dev/install.sh | bash -s -- --version v${CHAINLOOP_VERSION}
-          else
-            curl -sfL https://docs.chainloop.dev/install.sh | bash -s
-          fi
-          
+          bash -c "$(curl -fsSL https://raw.githubusercontent.com/chainloop-dev/labs/dev/tools/src/lib/chainloop.sh) ; install_chainloop_labs dev"
+          . ~/chainloop.sh
+          install_chainloop_cli_and_cosign
+
       - name: Download all workflow run artifacts
         uses: actions/download-artifact@v3
-      
+
       - name: Initialize Attestation
-        run: chainloop attestation init # --contract-revision 2
+        run: . ~/chainloop.sh && chainloop_attestation_init && ls -las
```

Comment:
I am not convinced about wrapping this command (`chainloop_attestation_init`) tbh.

Reply:
This about hiding the logic.

```

chainloop_attestation_init() {
  log "Initializing Chainloop Attestation"
  if [ -z "${CHAINLOOP_CONTRACT_REVISION+x}" ]; then
    chainloop attestation init -f 2>&1
  else
    chainloop attestation init -f --contract-revision ${CHAINLOOP_CONTRACT_REVISION} 2>&1
  fi
}
```

We can just include that bash code in the workflow but then improving this in the future will force us to update gitlab/jenkins/azure devops reusable workflows as well.